v2.0 compile error in test

On v2.0-branch, with `--enable-tests `, the tests fail to compile with GCC >= 6.  This is due to libva deprecated warnings and the tests are compiled with `-Wall -Werror`.  Older GCC versions do not appear to treat deprecated-declarations as errors even with `-Wall -Werror`.

```
g++ -DHAVE_CONFIG_H -I. -I..  -I/home/uaeoff/Work/workspace/media/install/include -I/home/uaeoff/Work/workspace/media/install/include -DGTEST_HAS_PTHREAD=1 -DGTEST_USE_OWN_TR1_TUPLE=0 -DGTEST_LANG_CXX11=1 -DGTEST_HAS_TR1_TUPLE=1 -DPTHREADS -DVA_DRIVERS_PATH="\"/home/uaeoff/Work/workspace/media/install/lib/dri\"" -std=c++11   -I/home/uaeoff/Work/workspace/media/install/include -Wall -Werror -I../test/gtest/include   -g3 -MT test_va_api-test_va_api_fixture.o -MD -MP -MF .deps/test_va_api-test_va_api_fixture.Tpo -c -o test_va_api-test_va_api_fixture.o `test -f 'test_va_api_fixture.cpp' || echo './'`test_va_api_fixture.cpp
In file included from test_va_api_fixture.h:27:0,
from test_va_api_fixture.cpp:25:
test_data.h:37:9: error: ‘VAProfileH264Baseline’ is deprecated [-Werror=deprecated-declarations]
VAProfileH264Baseline,
^~~~~~~~~~~~~~~~~~~~~
In file included from test_va_api_fixture.h:24:0,
from test_va_api_fixture.cpp:25:
/home/uaeoff/Work/workspace/media/install/include/va/va.h:327:5: note: declared here
VAProfileH264Baseline va_deprecated_enum = 5,
^~~~~~~~~~~~~~~~~~~~~
In file included from test_va_api_fixture.h:28:0,
from test_va_api_fixture.cpp:25:
test_streamable.h: In function ‘std::ostream& operator<<(std::ostream&, const VAProfile&)’:
test_streamable.h:73:10: error: ‘VAProfileH264Baseline’ is deprecated [-Werror=deprecated-declarations]
case VAProfileH264Baseline:
^~~~~~~~~~~~~~~~~~~~~
In file included from test_va_api_fixture.h:24:0,
from test_va_api_fixture.cpp:25:
/home/uaeoff/Work/workspace/media/install/include/va/va.h:327:5: note: declared here
VAProfileH264Baseline va_deprecated_enum = 5,
^~~~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
Makefile:635: recipe for target 'test_va_api-test_va_api_fixture.o' failed
make[2]: *** [test_va_api-test_va_api_fixture.o] Error 1
make[2]: Leaving directory '/home/uaeoff/Work/workspace/media/build/libva-utils/test'
Makefile:468: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/uaeoff/Work/workspace/media/build/libva-utils'
Makefile:400: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -36,7 +36,7 @@ before_install:
   - sudo apt-get install -y libxfixes-dev
 
 install:
-  - git clone https://github.com/01org/libva.git
+  - git clone -b v2.0-branch https://github.com/01org/libva.git
   - (cd libva && ./autogen.sh && ./configure --prefix=/usr && sudo make install)
 
 

--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -95,6 +95,7 @@ test_va_api_CPPFLAGS =						\
 
 test_va_api_CXXFLAGS =						\
 	-Wall -Werror						\
+	-Wno-deprecated-declarations				\
 	$(AM_CXXFLAGS)						\
 	$(NULL)
 

